handle_actions.go

```diff
@@ -75,7 +75,20 @@ func (a ActionReplyHTTP) Perform(context Context) error {
 		logrus.WithField("err", err).Error("failed to render template for http")
 		return err
 	}
-	return ec.Blob(a.StatusCode, contentType, []byte(msg))
+	err = ec.Blob(a.StatusCode, contentType, []byte(msg))
+	if err != nil {
+		return err
+	}
+
+	// finalize the HTTP response so that further actions make our response wait
```

Comment:
so previously, if not flushed, the connection blocks the thread from executing others?

Reply:
it sounds like it just sends the reply when this handler is actually ready. Extra sleeps might block for some time the response, is this correct?

Reply:
Yes, I think it was like this: 
1. echo calls our handler when we get an HTTP request 
2. reply_http sets the response body and status code 
3. other actions are run (such as `sleep: duration: 10s`), blocking the thread
4. when our handler returns, echo sends the response and closes the HTTP connection 

With the new changes, we just flush the HTTP connection during the `reply_http` action performer. Based on a comment David made somewhere, I'll check if we should manually set the Content-Length also here.